### PR TITLE
Search do not show the name written in the field

### PR DESCRIPTION
**Pull request by marcoAntonioNina**
## Issue & Reproduction Steps
Updating the query of the PmqlInput component placed the generated Pmql.

## Solution
- Add a parameter for update query.

## How to Test
Search the mentioned sections and also review the ticket https://processmaker.atlassian.net/browse/FOUR-14004

## Related Tickets & Packages
- [https://processmaker.atlassian.net/browse/FOUR-14030](FOUR-14030)
- - [https://processmaker.atlassian.net/browse/FOUR-14004](FOUR-14004)

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next
ci:package-savedsearch:observation/FOUR-14030
ci:deploy

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-3f438d0e91.engk8s.processmaker.net

File: resources/js/components/shared/PmqlInput.vue [hunk 1]
```diff
@@ -260,8 +261,12 @@ export default {
       this.calcInputHeight();
     },
     value() {
-      if (this.query !== this.value) {
-        this.query = this.value || "";
+      if (this.updateQuery) {
```
[the remark this answers is not in the document]
Comment (marcoAntonioNina) [reply resources/js/components/shared/PmqlInput.vue]: the change was made.

**Review by agustinbusso — APPROVED** [review]

**Review by boliviacoca — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-3f438d0e91.engk8s.processmaker.net

**Review by caleeli — APPROVED** [review]
Working as expected.
Checked Process, Request, Projects, PMBlock searches
Checker Charts in Screen Builder.

**Pull request merged**